app/controllers/admin/facilities_controller.rb

```diff
@@ -9,23 +9,35 @@ class Admin::FacilitiesController < AdminController
     :validate_facility_rows, if: :file_exists?, only: [:upload]
 
   def index
-    authorize([:manage, :facility, Facility])
+    unless Flipper.enabled?(:user_roles)
```

Comment:
We have to turn off all the Pundit checks if we want to user this new user role auth.